lnrpc/autopilotrpc/autopilot_server.go

```diff
@@ -51,6 +51,13 @@ var (
 	}
 )
 
+// RPCServer is shell struct holding a reference to the actual sub-server. It
+// is used to register the gRPC sub-server with the root server before we have
+// the necessary dependencies to populate the actual sub-server.
+type RPCServer struct {
```

Comment:
Could we perhaps name this `ServerShell` instead? Would make it a bit easier to distinguish from the actual server.

Reply:
Good idea, done!